github-373-pr: Added index for AWS Region and LZ terms (#379)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

*See also*: xref:ARM64[ARM64]

[[aws-local-zone]]
==== image:images/yes.png[yes] AWS Local Zone (noun)

> *Description*: An _Amazon Web Services (AWS) Local Zone_ represents a location that places cloud resources closer to a metropolitan AWS Region. An AWS Local Zone location is prepended by an AWS Region code that is followed by an identifier. For example `us-west-2-lax-1a`. 

Reviewer: ```suggestion
*Description*: An _Amazon Web Services (AWS) Local Zone_ represents a location that places cloud resources closer to a metropolitan AWS Region. An AWS Local Zone location is prepended by an AWS Region code that is followed by an identifier, such as `us-west-2-lax-1a`. 
```

Author: Thanks, @sbmetz . I agree. Updated.

*Incorrect forms*:

*See also*: xref:arm[Azure Resource Manager]

> [[aws-local-zone]]

Reviewer 2: These are out of order alphabetically - can you move them to the correct spots?

Author: Ah, yes. I was going by the tag, but it should be the word entry. Updated.

> ==== image:images/yes.png[yes] AWS Local Zone

Reviewer 2: This is missing the part of speech

*Description*: An _Amazon Web Services (AWS) Local Zone_ represents a location that places cloud resources closer to a metropolitan AWS Region. An AWS Local Zone location is prepended by an AWS Region code that is followed by an identifier, for example `us-west-2-lax-1a`. 

*Use it*: yes

*Incorrect forms*: Local Zone, LZ, local zone

*See also*: xref:opt-in[opt in], xref:aws-opt-in-region[AWS opt-in Region]

[[aws-opt-in-region]]

> ==== image:images/yes.png[yes] AWS opt-in Region

Reviewer 2: This is missing the part of speech

Author: Added "(noun)"
